packages/core/src/components/Dropdown/Dropdown.example.html

```diff
@@ -0,0 +1,25 @@
+<label class="ds-c-label ds-u-margin-top--0" for="options">Dropdown label</label>
```

Comment:
This is the same html example as the original one for <Select>, I just moved it to it’s own file for organization